src/theme/Navbar/MobileSidebar/Search/index.js

```diff
@@ -3,7 +3,6 @@ import React from 'react';
 import NavbarSearch from "@theme/Navbar/Search";
 import SearchBar from "@theme/SearchBar";
 import {useThemeConfig} from "@docusaurus/theme-common";
-import AIButton from "@theme/Navbar/AIButton";
 
 export default function NavbarMobileSidebarSearch() {
```

Comment:
There's a slight overlap with the mobile view and the light and dark mode toggle

<img width="430" alt="Screenshot 2024-11-28 at 12 40 47" src="https://github.com/user-attachments/assets/403de1a7-d5e2-4f15-a263-5c9cedcd3440">
<img width="377" alt="Screenshot 2024-11-28 at 12 42 12" src="https://github.com/user-attachments/assets/5d71e37d-0dbd-4c57-8548-009a0deffd66">

Reply:
Hello @AE-0h Wondering if you got a chance to look at this?

Reply:
Button placement/overlap issue with toggle is patched and was done through cms so no change to commit was required for that.  I updated the env var flow to use the docusaurus config.  I created a cookbook property and then call the value of that property in my AskCookbook component.  Please let me know if you require any further adjustments!

Reply:
Awesome. Thank you!